src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,27 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.Column;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
     private Long id;
+
+    @Column(name = "name", nullable = false)
     private String name;
+
+    @ManyToOne
+    @JoinColumn (name = "country_id")
     private Country country;
 
     public Actor() {
```

Comment:
Don't create a default constructor when it is not needed. Hibernate uses reflection and proxies to instantiate entities, so unless you have a specific reason for providing an empty constructor, it is not required.